amundsen_application/static/js/components/HomePage/index.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { RouteComponentProps } from 'react-router';
+
+// TODO: Use css-modules instead of 'import'
+import './styles.scss';
+
+import BookmarkList from 'components/common/Bookmark/BookmarkList';
+import PopularTables from 'components/common/PopularTables';
+import { SearchAllReset } from 'ducks/search/types';
+import { searchReset } from 'ducks/search/reducer';
+import SearchBar from 'components/SearchPage/SearchBar';
+
+
+export interface DispatchFromProps {
+  searchReset: () => SearchAllReset;
+}
+
+export type HomePageProps = DispatchFromProps & RouteComponentProps<any>;
+
+export class HomePage extends React.Component<HomePageProps> {
+  constructor(props) {
+    super(props);
+  }
+  
+  componentDidMount() {
+    this.props.searchReset();
+  }
+
+  render() {
+    return (
+      <>
```

Comment:
The wrapper `<>` element is useful when you have multiple "root" elements on a JSX node. For example
```
// Not allowed
(
  <h1> header </h1>
  <body> body</body>
)

// Allowed
(
  <>
    <h1> header </h1>
    <body> body</body>
  </>
)


```

Reply:
i.e. you don't need the `<>` here